### PR TITLE
At hr admin views all users

### PR DESCRIPTION
**Pull request by arbertheo**
we added the ability for the admin to view all users

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 class UsersController < ApplicationController
+  before_action :authorize_user
```
Comment (nvanselow) [inline app/controllers/users_controller.rb]: It looks like this code is replicated from the admin_users_controller. Does this need to be deleted?

File: config/routes.rb [hunk 2]
```diff
@@ -5,6 +5,10 @@
 
   resources :homepage, only: [:index]
 
+  resources :admin do
+    resources :users, except: [:create, :edit, :update, :new]
+  end
```
Comment (nvanselow) [inline config/routes.rb]: Can we switch this to "only" to match all our other routes?

File: spec/features/admin/admin_sees_all_users_spec.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'Admin views all venues' do
+  scenario 'When admin visits page sees all users.' do
+    users = FactoryGirl.create_list(:user, 3)
```
Comment (nvanselow) [inline spec/features/admin/admin_sees_all_users_spec.rb]: Let's also add a test where a non-admin attempts to access this page.

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (freyjasdottir) [inline app/controllers/users_controller.rb]: This is there intentionally. I don't think we want anyone who isn't authorized to be able to see a user list.

File: config/routes.rb (hunk 2 above)
Comment (freyjasdottir) [inline config/routes.rb]: Changed to `only: [:index]`

File: spec/features/admin/admin_sees_all_users_spec.rb (hunk 3 above)
Comment (freyjasdottir) [inline spec/features/admin/admin_sees_all_users_spec.rb]: Added

**Pull request merged**